scripts/tasks/backup

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Can we name this something with Postgres in the name. We may have multiple databases and need to distinguish which restore/backup script is for which database.

Reply:
changed to backup_postgres